contents/docs/surveys/troubleshooting.mdx

```diff
@@ -54,3 +54,14 @@ You can call `getActiveMatchingSurveys` in your browser console to see which sur
 posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
 ```
 
+## How are surveys priced?
+
+Surveys are priced based on the [number of responses](/pricing?product=surveys). However, because surveys is a tool that utilizes other parts of PostHog, such as events and feature flags, it's important to highlight the underlying costs too.
+
+Whenever a response is sent, it's stored as an event in PostHog, along with survey "shown" and "dismissed" events. Events are free up to the 1st million per month, and then $0.00031 after that. This should be negligible unless you're sending surveys to millions of users a month.
```

Comment:
we generally try to limit using actual pricing figures anywhere because they are liable to change. 

```suggestion
Surveys incur additional costs in two ways:

1. Events are stored when surveys are shown or dismissed, and when a response is submitted.
2. Feature flags are used when user property targeting or linked flags are used in a survey.

We have very generous free volumes for events and feature flags every month, and beyond the free tier costs per event or flag request are negligible. Check our our [pricing page](/pricing) for exact figures, but for 10,000 responses per month you can expect to pay less than $10 in other costs.

If you use feature flags outside of just surveys, you likely won't incur any additional costs, as flag API calls contain all flags needed for the session.
```

@joethreepwood can prob make that more succinct. And i'm not sure about the accuracy of the last sentence but i think that is largely true? or something along those lines.

Reply:
> If you use feature flags outside of just surveys, you likely won't incur any additional costs, as flag API calls contain all flags needed for the session.

It's priced based on decide requests so this would still incur additional costs if they used flags as part of feature flags right? Unless you meant something else?

Reply:
No additional costs _for surveys_

Reply:
that if you're already using feature flags, you wouldn't be charged extra for using surveys with feature flags. (since generally there are no extra decide requests being made)

Buuut, if you're not using feature flags, then there's an extra charge for using surveys with feature flags.

Reply:
errr I thought those scenarios were supposed to be flipped? Where if you have existing feature flags already, survey flags are charged because there's no way to differentiate the different types of flag requests. But if you don't use feature flags at all, then survey flags are NOT charged for

Reply:
Sorry, that was very poorly worded on my side, I was thinking of the linked flag case, but in that case it means you're already using flags, so it's wrong.

You're right, if you don't use flags, survey user targeting flags aren't charged.

But if you're already using flags, you're charged _ONLY_ if you make extra requests for flags for surveys, which doesn't happen by default.

Reply:
Well, since we decided to go with charging for dependent flags usage at all times, we can leave the last paragraph here out totally then!

Reply:
> If you use feature flags outside of surveys, you likely won't incur any additional flag costs, as flag API calls contain all flags needed for the session.

This is still valid though 😅 ; there's no extra charge if you already have flags, unless you make extra requests

Reply:
surveys can make extra requests though no?? Either way I think it's more confusing to have that statement than to not

Reply:
they don't currently, since `getFeatureFlag()` doesn't mean an extra request, and surveys are loaded after decide response.

I think it's useful to have this in because it reassures users that there's no exorbitant _extra_ fees for using flags with surveys. I'm not sure what the confusion is here, mind elaborating?

Reply:
As a user I feel like this statement `10,000 survey responses per month would typically cost less than $10 per month for events and flags, for example.` is what reassures me and explains the situation best already

With the last statement, do we mean something like "flags don't get double charged in feature flags pricing if they're used in surveys" <-- something like that?? That still sounds confusing to me

Reply:
Fair 👍

Reply:
we can always add it if people are confused!